schemas/dev/observation.schema.json

```diff
@@ -38,7 +38,20 @@
     "date": {
       "type": "string",
       "format": "date",
-      "description": "Date when observation was recorded"
+      "description": "Date of observation, or end date of observation period"
+    },
+    "start_date": {
+      "type": "string",
+      "format": "date",
+      "description": "Start date of observation duration"
+    },
+    "duration_type": {
+      "enum": [
+        "event",
+        "block"
+      ],
+      "default": "event",
+      "description": "Duration type, either present throughout the period (block), or something that occurred within the period (event)"
```

Comment:
Maybe we should force this to be 'event' if a start_date hasn't been specified for this specific observation (and we're taking the visit start date instead or something from a date hierarchy)? That way we can indicate that a 'block' type means we have confirmed start and end dates, rather than estimates.